### ISSUE TITLE
Snippet cache not cleared for all languages

### ISSUE THREAD

**Issue by acasar** [rainlab/pages-plugin#235]
When updating a static page (with Translate plugin installed), the snippet cache is cleared only for the current CMS language.

As a result snippets in other languages do not appear until I manually run ``php artisan cache:clear``.

I use the following code as a temporary fix:

```php
Event::listen('pages.object.save', function($controller, $object, $type) {
    if($type != 'page') return;

    $backupLocale = Lang::getLocale();
    $locales = Locale::listAvailable();
    $theme = Theme::getEditTheme();

    foreach($locales as $code => $name) {
        Lang::setLocale($code);
        StaticPage::clearMenuCache($theme);
        SnippetManager::clearCache($theme);
    }

    Lang::setLocale($backupLocale);
});
```

**Comment by munxar** [comment]
I can confirm this, this happens even if the translate plugin is not installed. My setup:
in app.php 'locale' => 'en' and backend language is set to 'de'.
changes in snippets are not reflected immediately after save. If backend language is set to 'en', changes in snippets are reflected after save.

**Comment by LukeTowers** [comment]
Fixed in #279
